src/aleph/http.clj

```diff
@@ -94,6 +95,7 @@
    | `stats-callback` | an optional callback which is invoked with a map of hosts onto usage statistics every ten seconds
    | `max-queue-size` | the maximum number of pending acquires from the pool that are allowed before `acquire` will start to throw a `java.util.concurrent.RejectedExecutionException`, defaults to `65536`
    | `control-period` | the interval, in milliseconds, between use of the controller to adjust the size of the pool, defaults to `60000`
+   | `dns-options` | an optional map with async DNS resolver settings, for more information check `aleph.netty/create-dns-resolver`. When set, ignores `name-resolver` setting from `connection-options` in favor of shared DNS resolver instace
```

Comment:
I think this should be just `dns-resolver`

Reply:
@ztellman Do you mean to rename `dns-options` to `dns-resolver`?

Reply:
@ztellman 👍